src/pypdfium2/_helpers/textpage.py

```diff
@@ -130,29 +104,18 @@ def get_text_bounded(self, left=None, bottom=None, right=None, top=None, errors=
         return buffer.raw.decode("utf-16-le", errors=errors)
     
     
-    def get_text(self, *args, **kwargs):
-        """
-        Deprecated alias for :meth:`.get_text_bounded`. Will be removed with the next major release.
-        """
-        return self.get_text_bounded(*args, **kwargs)
-    
-    
-    def count_rects(self, index=0, count=0):
+    def count_rects(self, index=0, count=-1):
         """
         Parameters:
-            index (int): Character index at which to start.
-            count (int): Character count to consider (defaults to :meth:`.count_chars`).
+            index (int): Start character index.
+            count (int): Character count to consider (defaults to -1 for all remaining).
         Returns:
-            int: The number of text rectangles on the page.
+            int: The number of text rectangles for the given character range.
         """
-        
-        if self.n_chars == 0:
-            return 0
-        if count == 0:
-            count = self.n_chars
```

Comment:
Should have been `self.n_chars - index` here, but it's more convenient to let PDFium do this, anyway.

TODO: update other functions from `count=0` to `count=-1`.